sdk/eventhub/azure-eventhub/azure/eventhub/_consumer.py

```diff
@@ -89,6 +89,7 @@ def __init__(
             "track_last_enqueued_event_properties", False
         )
         idle_timeout = kwargs.get("idle_timeout", None)
+        self._use_tls = kwargs.get("use_tls", True)
```

Comment:
It looks like we're setting a default of True for this everywhere. Can we just do a default at the highest level (ConsumerClient/in config I believe + the highest pyamqp level), then require it everywhere else? Wondering if it might be easier to track.

Reply:
Maybe at the pyAMQP Connection level where the client wants an insecure_connection, which is then passed to the transport.